README.md

```diff
@@ -14,87 +14,44 @@ This project offers environment for all the labs in **Red Hat RHCSA/RHCE 7 Cert
 - [Vagrant](https://www.vagrantup.com): tool for building complete development environments. With an easy-to-use workflow and focus on automation
 
 Don't worry you don't have to be expert on any of these tools.
-### 1. Install virtualbox :
 
-**1.1 Install virtualbox on Ubuntu:**
+## install Vagrant
```

Comment:
heading should be `Install Dependencies` not `install Vagrant` because we  are install both vagrant and virtualization hypervisors either kvm or virtualbox & the section above it in `README.md` should be called Dependencies.